shared/src/main/scala/io/github/hamsters/Show.scala

```diff
@@ -50,6 +50,10 @@ object Showable {
   implicit val unitShowable: Showable[Unit] = new Showable[Unit] {
     override def format(value: Unit) = value.toString
   }
+
+  def fromToStringLowerCase[A]: Showable[A] = new Showable[A] {
```

Comment:
why is it called `fromString`, as it takes a A, not a String?

Reply:
It seemed like a good idea at the moment.
I did not see the method just above (I must be blind ...)

It's something like this one from cats : 
https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/Show.scala#L42